test/PR_test/integration_test/test_pipeline.py

```diff
@@ -314,6 +317,21 @@ def test_pipeline_benchmark_smoke(self):
                 except:
                     self.fail("exception occurred")
 
+    def test_multi_ds(self):
+        ds = {"ds_1": self.sample_torch_dataset, "ds_2": self.sample_torch_dataset}
+        pipeline = fe.Pipeline(train_data=ds)
+        pipeline.benchmark()
+
+    def test_multi_train(self):
+        ds = {"ds_1": self.sample_torch_dataset, "ds_2": self.sample_torch_dataset}
+        pipeline = fe.Pipeline(eval_data=ds)
+        pipeline.benchmark(mode="eval")
+
+    def test_multi_test(self):
+        ds = {"ds_1": self.sample_torch_dataset, "ds_2": self.sample_torch_dataset}
```

Comment:
tests w/ scheduling would be good to verify that's working

Reply:
added the test